processor/framework.py

```diff
@@ -330,21 +353,17 @@ def get_submission_os(self):
         if distro == "CentOS":
             if os_version[0] == "7":
                 image_name = "centos7"
-        elif distro == "RedHatEnterprise" or distro == "AlmaLinux":
-            if os_version[0] == "8":
-                image_name = "centos8"
-            elif os_version[0] == "9":
+        elif distro == "RedHatEnterprise" or distro == "Alma":
+            if os_version[0] == "9":
                 image_name = "rhel9"
         elif distro == "Ubuntu":
-            if os_version[0:2] == "20":
-                image_name = "ubuntu2004"
-            elif os_version[0:2] == "22":
+            if os_version[0:2] == "22":
                 image_name = "ubuntu2204"
         else:
             raise Exception(
-                f"Unknown OS {distro} {os_version}, CROWN will not run without changes"
+                f"Unknown OS {distro} {os_version}, KingMaker will not run without changes"
             )
-        image = f"ghcr.io/kit-cms/kingmaker-images-{image_name}-{str(self.ENV_NAME).lower()}:main"
+        image = f"ghcr.io/kit-cms/kingmaker-images-{image_name}-{str(self.ENV_NAME).lower()}:update_env_PR"
```

Comment:
to be switched to main when merging

Reply:
This should only be changed back after the related kingmaker-images PR is done

Reply:
Covered by https://github.com/KIT-CMS/KingMaker/commit/7071854b5c9d885bbebdb79d24897ecf3cebf6d0